### PR TITLE
About us

### PR DESCRIPTION
**Pull request by AnisioMandlate**
Development of the About Page. Please review it and give your feedback

### CODE REVIEW

File: src/components/About Card/styles.module.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  align-items: center;
+  justify-content: space-between;
+  border: 1px solid rgba(23, 41, 49, 0.56);
+  border-radius: 16px;
+  padding: 40px 20px;
+  filter: drop-shadow(2px, 4px 12px rgba(0, 0, 0, 0.15));
+}
+
+.image {
+  /* max-width: 100px; */
```
Comment (adilsonmandlate) [inline src/components/About Card/styles.module.css:14]: Remove unused code

File: src/components/About Card/index.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from "react"
```
Comment (adilsonmandlate) [inline src/components/About Card/index.jsx:1]: For multi-worded names, avoid the use of spaces. Use instead PascalCase or camelCase

File: src/components/About Card/styles.module.css (hunk 1 above)
Comment (AnisioMandlate) [reply src/components/About Card/styles.module.css]: Done

File: src/components/About Card/index.jsx (hunk 2 above)
Comment (AnisioMandlate) [reply src/components/About Card/index.jsx]: Alright. Done

**Pull request merged**